inscriptions/ISic003979.xml

```diff
@@ -182,19 +188,16 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI334466">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#SS">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Ἀρχαγάθο
-                    <supplied reason="lost">υ</supplied>
-                    <surplus>²⁷Ἀρχάγαθο[ς]</surplus>²⁷<lb n="2"/>Μόσχου.
-                    <lb n="3"/>
+                    <lb n="1"/>Ἀρχαγάθο<supplied reason="lost"><app type="alternative"><rdg>υ</rdg><rdg>ς</rdg></app></supplied> <!-- two equally valid readings possible -->
```

Comment:
Yes, we do need to figure out apparatus. This wasn't flagged in the edition apparatus, so I put it in-line rather than down in the app but can revise, of course.

Reply:
Will try the `<certainty>` option here as well.